package.json

```diff
@@ -1549,7 +1542,7 @@
                     "type": "boolean",
                     "default": true,
                     "description": "%jupyter.configuration.jupyter.askForKernelRestart.description%",
-                    "scope": "resource"
+                    "scope": "application"
```

Comment:
Also seems like it might make sense as a workspace setting

Reply:
i don't see why one would want to get prompted to restart kernels in one workspace and not the other. 

agreed existing data should be moved to application level of we change this

Reply:
But someone might want to do that, and if it's working fine, why restrict it? Could be a team preference that they want to enforce, could be that someone has a different workflow in two projects and just finds that useful.

Reply:
Got it, thanks